.github/workflows/merge-commits.yml

```diff
@@ -25,16 +25,14 @@ jobs:
             const queueName = `${{ secrets.AZURE_QUEUE_NAME }}`;
             const queueServiceClient = QueueServiceClient.fromConnectionString(connectionString);
             const queueClient = queueServiceClient.getQueueClient(queueName);
-            var receivedMessages = await queueClient.receiveMessages();
-            const message = receivedMessages.receivedMessageItems[0];
-            const firstMessage = JSON.parse(message.messageText);
-            if(firstMessage === undefined) { return 0; }
-            const updateMsg = { "sha": firstMessage.sha, "author": firstMessage.author, "label": "in-progress" }
-            await queueClient.updateMessage(
-                message.messageId,
-                message.popReceipt,
-                JSON.stringify(updateMsg)
-            );
+            // Peek at messages in the queue
+            const peekedMessages = await queueClient.peekMessages({ numberOfMessages: 5 });
```

Comment:
Could the number of commits to batch be a variable?

Reply:
One of the ways is to add it as an env var and pass it. But we might have the same problem as we had with passing TARGET_BRANCH as env var.